### DISCUSSION TITLE
Changing the Navigation Bar

### DISCUSSION THREAD

**Discussion by dash622** [cotes2020/jekyll-theme-chirpy#1937] [Q&A]
### Checklist

- [X] I have read and accepted the [contributing guidelines](https://github.com/cotes2020/jekyll-theme-chirpy/blob/master/docs/CONTRIBUTING.md).

### How did you create the site?

Generated from `chirpy-starter`

### Description

I am looking to change the navigation bar on the left. Is there an easy way to do this?

### Operations you have already tried

I have looked through the main contact.yml file and can't find anything there. I attempted to try and edit the _site/index.html and it only worked for the main page. Will I have to edit the index.html for each individual page in order to make it work universally?

**Comment by kungfux** [comment]
It's not clear what exactly you have in mind for change, so it's hard to be prescriptive.

**Reply by dash622** [reply]
Just looking to move the "about me" portion to be the top tab rather than at the bottom.

**Reply by kungfux** [reply]
You can find tab definitions under `_tabs` folder. To change their order, you need to change the `order` value in the front matter for every tab.

```markdown
---
order: 4
---
```

However, there is no `Home` tab available, so `About` tab will stay under the `Home` tab. If you want to go beyond, you will have to override the `_includes/sidebar.html` by copying it from this repository and placing it at the same location in your repository, then change the [logic](https://github.com/cotes2020/jekyll-theme-chirpy/blob/master/_includes/sidebar.html#L23) to meet your expectations.

**Comment by SoumyaK4** [comment]
#2099 check this
